lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -716,7 +716,7 @@ namespace Js
             simdFloat32x4TypeDynamic = DynamicType::New(scriptContext, TypeIds_SIMDObject, simdFloat32x4Prototype, nullptr, NullTypeHandler<false>::GetDefaultInstance(), true, true);
 
             simdFloat32x4TypeStatic = StaticType::New(scriptContext, TypeIds_SIMDFloat32x4, simdFloat32x4Prototype , nullptr);
-            simdFloat64x2TypeStatic = StaticType::New(scriptContext, TypeIds_SIMDFloat64x2, simdFloat64x2Prototype, nullptr);
+            //simdFloat64x2TypeStatic = StaticType::New(scriptContext, TypeIds_SIMDFloat64x2, simdFloat64x2Prototype, nullptr);
```

Comment:
@abchatra cud u kindly advise if "a disabled config key" means an actual runtime flag in `GetScriptContext()->GetConfig()` or macro guard `#ifdef SIMD_FLOAT64x2` like @nmostafa is suggesting.